src/gen/libhighs.jl

```diff
@@ -13,18 +13,30 @@ struct HighsCallbackDataOut
     running_time::Cdouble
     simplex_iteration_count::HighsInt
     ipm_iteration_count::HighsInt
+    pdlp_iteration_count::HighsInt
```

Comment:
cc @jajhall adding a field to the middle of a struct is technically breaking, but I don't think it really matters in this case because no one was really using the callbacks yet.

Reply:
OK, fair cop, well spotted!